app/hooks/useAuthorizedSettingsConfig.ts

```diff
@@ -170,6 +172,14 @@ const useAuthorizedSettingsConfig = () => {
         group: t("Integrations"),
         icon: WebhooksIcon,
       },
+      Diagrams: {
```

Comment:
I was half way through deciding when handed over to you, lets just rename all these to `Drawio` instead of `Diagrams`? I

Reply:
Sure!

Reply:
Fixed!